packages/cli/src/commands/pg/wait.ts

```diff
@@ -39,6 +39,7 @@ export default class Wait extends Command {
       let status
       let waiting = false
       let retries = 20
+      const notFoundMessage = 'Waiting for provisioning to start...'
```

Comment:
```suggestion
      const notFoundMessage = 'Waiting to provision...'
```
CX review suggestion @justinwilaby